src/main/scala/bank/Main.scala

```diff
@@ -1,7 +1,15 @@
 package bank
 
+import akka.actor._
+import akka.http.scaladsl.Http
+
 object Main extends App {
 
   println("Welcome to Banking App")
+  val controller = new Controller
+
+  implicit val actorSystem = ActorSystem()
+
+  Http().newServerAt("localhost", 9000).bind(controller.allRoutes)
```

Comment:
please make host and port configurable

Reply:
used typesafe config library and make host and port configurable in application.conf via environment variables